source/plugins/ruby/in_kube_nodes.rb

```diff
@@ -47,7 +47,7 @@ def initialize(kubernetesApiClient = nil,
       @@rsPromMonitorPodsLabelSelectorLength = @env["TELEMETRY_RS_PROM_LABEL_SELECTOR_LENGTH"]
       @@rsPromMonitorPodsFieldSelectorLength = @env["TELEMETRY_RS_PROM_FIELD_SELECTOR_LENGTH"]
       @@collectAllKubeEvents = @env["AZMON_CLUSTER_COLLECT_ALL_KUBE_EVENTS"]
-      @@osmNamespaceCount = @env["TELEMETRY_OSM_CONFIGURATION_NAMESPACES_COUNT"]
+      @@osmNamespaceCount = @env["OSM_CONFIGURATION_NAMESPACES_COUNT"]
 
       @ContainerNodeInventoryTag = "oneagent.containerInsights.CONTAINER_NODE_INVENTORY_BLOB"
       @insightsMetricsTag = "oneagent.containerInsights.INSIGHTS_METRICS_BLOB"
```

Comment:
there is new file with name "sudo" added. can you please revert that since its not related?

Reply:
Reverted